.github/workflows/build-tarball.yml

```diff
@@ -12,8 +12,7 @@ on:
       - '!.github/workflows/build-tarball.yml'
   push:
     branches:
-      - master
-      - main
+      - $default-branch
```

Comment:
Is this a thing? Wouldn't that target a branch named `$default-branch`?

Reply:
yes, see https://github.blog/changelog/2020-07-22-github-actions-better-support-for-alternative-default-branch-names/ and [here](https://github.com/actions/starter-workflows/search?q=%24default-branch) for example use

Reply:
AFAIK it only works in templates. It is replaced with the actual default branch name when a workflow is created from the template.

Reply:
@targos yes, I revert the use of `$default-branch`, It wont work for standard workflow.